website/content/api-docs/system/plugins-catalog.mdx

```diff
@@ -44,6 +44,32 @@ $ curl \
             "mysql-database-plugin",
             "postgresql-database-plugin"
         ],
+        "detailed": [
```

Comment:
I think this shows up as "details"?

```suggestion
        "details": [
```

Reply:
I checked this and the API actually has it as `detailed`, and then the CLI transforms it to `details`. I'm not sure why we did that, consistency would have been better :(

Reply:
Oh wow you're right, the CLI even transforms it when specifying -format=json :(